fastlane/README.md

```diff
@@ -242,6 +242,14 @@ Creates a new PR after new snapshot files were generated
 
 Creates a new PR on purchases-ios-snapshots after new snapshot files were generated
 
+### ios enable_customer_center
+
+```sh
+[bundle exec] fastlane ios enable_customer_center
+```
+
+Enable Customer Center
+
```

Comment:
Does this lane exist? I can't seem to find it in the Fastfile.

Reply:
oops, no, not sure why it got added, I'll remove